polaris-core/src/main/java/org/apache/polaris/core/entity/PolarisEntity.java

```diff
@@ -182,6 +182,12 @@ public static List<NameAndId> toNameAndIdList(List<EntityNameLookupRecord> entit
         .orElse(null);
   }
 
+  public static boolean isFederated(PolarisBaseEntity entity) {
```

Comment:
If only a subset of entity types support this, I don't think it should be present in the base type `PolarisEntity`

Reply:
I tend to agree with @eric-maynard 's point here

Reply:
Not sure why this comment's resolved.

I also agree that this attribute is, if at all, only need of a subset of all types and should only live there.

Reply:
I'd resolved it because I'd fixed it in last revision. But I didn't realize my push hadn't gone through - I have ssh cert issues sometimes in my terminal 🤷🏽‍♂️